Tests/ScopeCheck/operators.silt

```diff
@@ -11,6 +11,10 @@ data Bool : Type where
   false : Bool
   true  : Bool
 
+data List (A : Type) : Type where
```

Comment:
Could you split this change and the `DeclaredRecord` one off into a smaller PR to unclutterer this?

Reply:
I got rid of the `DeclaredRecord` thing overall because I didn't need it. And I can get rid of this change, I also didn't need this.

Reply:
I think it's cleaner than having 4 unnamed parameters in an enum.